sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -127,6 +127,10 @@ trait CheckAnalysis extends PredicateHelper with LookupCatalog {
       case CacheTable(u: UnresolvedRelation, _, _, _) =>
         failAnalysis(s"Table or view not found for `CACHE TABLE`: ${u.multipartIdentifier.quoted}")
 
+      case UncacheTable(u: UnresolvedRelation, _, _) =>
+        failAnalysis(
+          s"Table or view not found for `UNCACHE TABLE`: ${u.multipartIdentifier.quoted}")
```

Comment:
After a second look, I think it's better to be consistent with INSERT and just say `Table or view not found: xxx`. When people run the command, they definitely know which command triggers the table not  found issue.

Reply:
OK. I will revert other commands as well in a separate PR.